lib/koans/11_processes.ex

```diff
@@ -1,22 +1,22 @@
 defmodule Processes do
   use Koans
 
-  koan "tests run in a process!" do
+  koan "Tests run in a process" do
     assert Process.alive?(:__)
```

Comment:
Hm... do you think this is too hard to figure out for the first process koan? Without looking at the answer I guess that the blank expects `self()` but I'm not sure.